templates/Dataland/v2/index.html.twig

```diff
@@ -0,0 +1,111 @@
+{% extends 'base.html.twig' %}
+{% block title %}{{ dataset.udi }}{% endblock %}
+
+{% block head %}
+    {{ parent() }}
+
+    {% if dataset.isAvailable %}
+        {% include 'Dataland/metadata.markup.twig' %}
+    {% else %}
+        <meta name="robots" content="noindex"/>
+    {% endif %}
+{% endblock %}
+
+{% block stylesheets %}
+    {{ parent() }}
+    {{ encore_entry_link_tags('data-land') }}
+{% endblock stylesheets %}
+
+{% block body %}
+    <div class="bootstrap">
+        <div class="container">
+            <div class="row">
+                <main class="col-lg-9 overflow-auto">
+                    <div class="pt-3">
+                        {% include 'Dataland/v2/status-badge.html.twig' %}
+                        <h2> {{ dataset.title }} </h2>
+                    </div>
+                    <div class="d-flex justify-content-between py-2">
+                        <div>
+                            <div>
+                                <strong>Authors:</strong> {{ dataset.datasetSubmission.authors }}
+                            </div>
+                            <div>
+                                <strong>Published on </strong> {{ dataset.acceptedDate|date('M d Y H:i T', 'UTC') }}
+                            </div>
+                        </div>
+                        <div>
+                            <div>
+                                <div>
+                                    <strong>DOI:</strong> {{ dataset.doi.doi }}
+                                </div>
+                                <div>
+                                    <strong>UDI:</strong> {{ dataset.udi }}
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <hr>
+                    <div class="py-2">
+                        <h6><strong>Abstract:</strong></h6>
+                        <p>
+                            {{ dataset.abstract }}
+                        </p>
+                    </div>
+                    <hr>
+                    <div class="py-2">
+                        <div id="dlolmap" style="width: 100%;height:480px;"
```

Comment:
I see this hardcoded here, yet when testing see different values. I'm assuming something overwrites it, but where/what?